src/Platform/Microsoft.Testing.Platform/ServerMode/DotnetTest/IPC/ObjectFieldIds.cs

```diff
@@ -66,19 +66,25 @@ internal static class SuccessfulTestResultMessageFieldsId
     public const ushort Uid = 1;
     public const ushort DisplayName = 2;
     public const ushort State = 3;
-    public const ushort Reason = 4;
-    public const ushort SessionUid = 5;
+    public const ushort Duration = 4;
```

Comment:
isn't this a bad idea for backwards compatibility?Normally you want to add new data fields to the to the end, so you can read what you know, and skip the rest. BUT I don't know how your protocol solves this, so please feel free to ignore me / educate me, if I am wrong :)

Reply:
We still haven't shipped so this won't affect anything for now.

Reply:
I am correct here, but the api is not shipped so we can play around with the fields as we like.

Reply:
Yeah, exactly.